android/build.gradle

```diff
@@ -1,6 +1,8 @@
 buildscript {
   // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-  def kotlin_version = rootProject.ext.has('kotlinVersion') ? rootProject.ext.get('kotlinVersion') : project.properties['HeychargeSdk_kotlinVersion']
```

Comment:
Do not change that, leave it as it is, just check every kotlinVersion and ensure it's the same.